https://bugs.gentoo.org/911703
https://github.com/wxWidgets/Phoenix/pull/2441

From aeb557d01e7cd37176ebbf0f1ae6d0b53c115378 Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Tue, 25 Jul 2023 12:42:32 -0400
Subject: [PATCH] Fix compatibility with Cython 3.0.0

Remove unused import which seems to be removed in Cython 3.
--- a/wx/svg/_nanosvg.pyx
+++ b/wx/svg/_nanosvg.pyx
@@ -42,7 +42,6 @@ for manipulating the SVG shape info in memory.
 
 import sys
 
-cimport cython.object
 from cpython.buffer cimport (
     Py_buffer, PyObject_CheckBuffer, PyObject_GetBuffer, PyBUF_SIMPLE,
     PyBuffer_Release)
